values/samples/sample1-stream-to-nfs-job.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+appParameters:
+  # Files will be created and written with this interval.
+  checkpointIntervalMs: "10000"
+  checkpointTimeoutMs: "3600000"  # 1 hr
+  input-stream: "sample1"
+  #input-startAtTail: "false"
+  #input-startStreamCut: "UNBOUNDED"
+  output: "/mnt/examples-sample"
+  #parallelism: "1"
+imageRef:
+  name: "1.10.0-2.12-hadoop2.8.3"
+clusterConfiguration:
+  taskmanager.memory.jvm-metaspace.size: "536870912"  # 512 MiB
```

Comment:
Add the volumeMounts to this file instead.

Reply:
added